<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScalesServerClass</class>
 <widget class="QMainWindow" name="ScalesServerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>801</width>
    <height>511</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ScalesServer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QFrame" name="frame">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <layout class="QFormLayout" name="formLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="styleSheet">
              <string notr="true">font: 75 14pt &quot;MS Shell Dlg 2&quot;;</string>
             </property>
             <property name="text">
              <string>IP scales</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="le_ip_scales"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_2">
             <property name="styleSheet">
              <string notr="true">font: 75 14pt &quot;MS Shell Dlg 2&quot;;</string>
             </property>
             <property name="text">
              <string>Port scales</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="le_port_scales">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QCheckBox" name="chk_parse_data">
             <property name="styleSheet">
              <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
             </property>
             <property name="text">
              <string>Parse data</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0" colspan="2">
            <widget class="QPushButton" name="btn_connect_scales">
             <property name="styleSheet">
              <string notr="true">font: 75 13pt &quot;MS Shell Dlg 2&quot;;</string>
             </property>
             <property name="text">
              <string>Connect</string>
             </property>
            </widget>
           </item>
           <item row="5" column="0" colspan="2">
            <widget class="QPushButton" name="btn_disconnect_scales">
             <property name="styleSheet">
              <string notr="true">font: 75 13pt &quot;MS Shell Dlg 2&quot;;</string>
             </property>
             <property name="text">
              <string>Disconnect</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0" colspan="2">
            <widget class="QComboBox" name="cbox_scales_type">
             <property name="styleSheet">
              <string notr="true">font: 75 13pt &quot;MS Shell Dlg 2&quot;;</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QTextBrowser" name="brs_log"/>
      </item>
      <item row="1" column="0">
       <widget class="QFrame" name="frame">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <layout class="QFormLayout" name="formLayout_2">
           <item row="0" column="0">
            <widget class="QLabel" name="label_3">
             <property name="styleSheet">
              <string notr="true">font: 75 14pt &quot;MS Shell Dlg 2&quot;;</string>
             </property>
             <property name="text">
              <string>Port server</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="le_port_server">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="1" column="0" colspan="2">
            <widget class="QPushButton" name="btn_start_server">
             <property name="styleSheet">
              <string notr="true">font: 75 13pt &quot;MS Shell Dlg 2&quot;;</string>
             </property>
             <property name="text">
              <string>Start server</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0" colspan="2">
            <widget class="QPushButton" name="btn_stop_server">
             <property name="styleSheet">
              <string notr="true">font: 75 13pt &quot;MS Shell Dlg 2&quot;;</string>
             </property>
             <property name="text">
              <string>Stop server</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QTextBrowser" name="brs_log_scales"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>801</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="scalesserver.qrc"/>
 </resources>
 <connections/>
</ui>
